packages/create-vite-app/template/src/main.tsx

```diff
@@ -0,0 +1,34 @@
+import { isValidAutomergeUrl, Repo } from '@automerge/automerge-repo'
+import { BrowserWebSocketClientAdapter } from '@automerge/automerge-repo-network-websocket'
+import { IndexedDBStorageAdapter } from "@automerge/automerge-repo-storage-indexeddb"
+import { RepoContext } from '@automerge/automerge-repo-react-hooks'
+import {next as A} from "@automerge/automerge" //why `next`? See the the "next" section of the conceptual overview
```

Comment:
this is misformatted; probably you should make sure prettier has run over the lot of it. also, the comment is not very helpful: we should link directly to the docs.